_emails/oakland.md

```diff
@@ -1,41 +1,40 @@
 ---
 title: Oakland, CA
-permalink: "/oakland"
+permalink: "/defundopd"
```

Comment:
Any concerns that changing the permalink might affect those who are already sharing the /oakland URL?

Reply:
It may be better to stick to the style of other permalinks as well

Reply:
Yeah. Any links that are live cannot be changed at the moment. I will create an issue for this. We should work on a solution for redirects if we want to modify existing links.

Reply:
Good thought! Changed it back :)